examples/suppression_list/list_suppression_entries.py

```diff
@@ -0,0 +1,9 @@
+from sparkpost import SparkPost
+
+sp = SparkPost('YOUR API KEY')
+results = sp.suppression_list.list(
+    From='2015-05-07T00:00:00+0000',
```

Comment:
if it's because from is a keyword, I would change the the names of the params - `from_date`, `to_date`.